ansible/roles/girder-histomicstk/tasks/main.yml

```diff
@@ -1,61 +1,43 @@
 ---
 - name: NodeJS | Add PPA key
   command: "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68576280"
-  become: true
+  become: yes
+  become_user: root
 
 - name: NodeJS | Add PPA
   apt_repository:
-    repo: "deb https://deb.nodesource.com/node_6.x {{ ansible_distribution_release }} main"
-  become: true
+    repo: "deb https://deb.nodesource.com/node_4.x {{ ansible_distribution_release }} main"
```

Comment:
Is there a reason to use node 4 rather than node 6?  In the grider readthedocs we show using node 6 (http://girder.readthedocs.io/en/latest/prerequisites.html)